frontend/pages/sites/SiteListItem.test.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { MemoryRouter } from 'react-router-dom';
+
+import SiteListItem from './SiteListItem';
+// import GitHubLink from '@shared/GitHubLink';
+// import PublishedState from './PublishedState';
+
+const mockOrganization = {
```

Comment:
should these be fixtures in `/test/frontend/support/data/` ?

Reply:
Yes and you can update the `test/frontend/support/data/organizations.js` function to include the additional props.

Reply:
Done